### PR TITLE
Add files for lesson 7

### PR DESCRIPTION
**Pull request by Resires**

### CODE REVIEW

File: students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_1/bullet_point_adder.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#! python3
```
Comment (jedzej) [inline students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_1/bullet_point_adder.py]: Does this shebang really work? I see 2 potential problems here:
- space between `#!` and the path
- no full path
I'm used to following form of shebang working on unix-like systems `#!/usr/bin/python3`. On windows hosts shebang doesn't work anyway.

File: students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_1/bullet_point_adder.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# bullet_point_adder.py - Adds Wikipedia bullet points to the start
+# of each line of text on the clipboard.
+
+import pyperclip
+text = pyperclip.paste()
+# text = 'Lists of animals\nLists of aquarium life\nLists of biologists by author abbreviation\nLists of cultivars'
```
Comment (jedzej) [inline students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_1/bullet_point_adder.py]: remove commented code

File: students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_1/table_printer.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+def print_table(table):
+    columns_quantity = len(table)
+    rows_quantity = len(table[0])
+    column_widths = [0] * columns_quantity
+    for column in range(columns_quantity):
+        column_widths[column] = len(max(table[column], key=len))
```
Comment (jedzej) [inline students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_1/table_printer.py]: you don't really need to preallocate the `column_widths`
```
column_widths = [len(max(column, key=len)) for column in table]
```

also `map` can be used to produce list of lengths and pick max without extra keying
```
column_widths = [max(map(len, column)) for column in table]
```

File: students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_2/replace_the_substring.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+message = "Given a string. Replace in this string all the numbers 1 by the word one. Even if it is 11."
+print("Original message =", message)
+while message.find("1") != -1:
```
Comment (jedzej) [inline students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_2/replace_the_substring.py]: why not this? https://docs.python.org/3/library/stdtypes.html#str.replace

File: students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_2/to_swap_the_two_words.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+two_word_message = "Run Forrest"
+space_position = two_word_message.find(" ")
```
Comment (jedzej) [inline students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_2/to_swap_the_two_words.py]: Why not this? https://docs.python.org/3/library/stdtypes.html#str.split

**Review by jedzej — CHANGES_REQUESTED** [review]
The only coding problem here is that you've been using already acquired skills for iterables (index hunting and list slicing) instead of going through the docs and checking what Python has to offer when it comes specifically to string manipulation.
Please go through `str` built-in type documentation to get familiar with the toolkit python provides. 
https://docs.python.org/3/library/stdtypes.html#str

The rest is fine and solid

File: students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_2/to_swap_the_two_words.py (hunk 5 above)
Comment (Resires) [reply students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_2/to_swap_the_two_words.py]: It was specified in the task to use "count" method.
"Given a string consisting of words separated by spaces. Determine how many words it has. To solve the problem, use the method count."

File: students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_2/to_swap_the_two_words.py [hunk 6]
```diff
@@ -0,0 +1,5 @@
+two_word_message = "Run Forrest"
+list_of_words = two_word_message.split()
```
Comment (jedzej) [inline students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_2/to_swap_the_two_words.py:2]: This is fine. Consider also following options

If you plan to use only first 2 elements of the list you can assign them specifically on split:
```
first_word, second_word = two_word_message.split()
```

Alternatively you could solve this with a one-liner that'd work for longer strings and won't raise an exception for empty or single word strings:
```
" ".join(reversed(two_word_message.split()))
```

**Review by jedzej — APPROVED** [review]
Well done. Now it utilizes string features. You're good to merge!

**Pull request merged**